app/templates/events/trackVolunteers.html

```diff
@@ -50,7 +50,7 @@ <h1 class="text-center">{{event.name}}</h1>
   </div>
 </div>
 <form method="post">
-  <label class='d-none' id="eventLength">{{eventLength}}</label>
+  <label class='d-none'>{{eventLength}}</label>
```

Comment:
rather than just remove the id, we should clean up the other duplicate code around it and make sure it still works. Both the label and input after it are duplicated elsewhere. Do we know if the hidden elements inside the form are needed or the ones outside?